proxy/hdrs/MIME.cc

```diff
@@ -2381,7 +2381,7 @@ MIMEScanner::get(TextView &input, TextView &output, bool &output_shares_input, b
       } else {
         // This really should be an error (spec doesn't permit lone CR) but the regression tests
         // require it.
-        this->append({&RAW_CR, 1});
```

Comment:
As this change is purely to make a specific compiler happy, it should certainly have a comment to that effect. E.g. "// ICC 9.1 requires the explicit class name"

Reply:
I added a comment for you AMC! Your the best!